documentation/Triggers-and-Actions.md

```diff
@@ -41,10 +41,9 @@ Actions also return an **output** object (e.g. `createUser` Action returns user
 | `onApplicationSubmit`    | Postgres trigger on `Application` table                          |
 | `onApplicationSave`      | Postgres trigger on `Application` table                          |
 | `onApplicationWithdrawn` | Postgres trigger on `Application` table                          |
-| `onReviewStart`          | Postgres trigger on `Review` table                               |
+| `onReviewCreate`         | Postgres trigger on `Review` table                               |
 | `onReviewSubmit`         | Postgres trigger on `Review` table                               |
 | `onEditComment`          | Postgres trigger on `Review_response` table                      |
-| `onReviewSave`           | Postgres trigger on `Review` table                               |
 | `onApplicationAssign`    | Postgres trigger on `Review_section_assign` table                |
 | `onApprovalSubmit`       | Postgres trigger on `Review_section` table                       |
 | `onActionComplete`       | Postgres trigger on `Action_queue` table                         |
```

Comment:
Not your issue, but this doesn't exist any more (it was an idea for sequential actions that has been made obsolete).
Can delete line.

Reply:
23ca77b50f1ed895a2209e39dd4fd71b0b4af79b